content/team/collaboration.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Collaboration"
+date: 2021-06-28T13:44:15-05:00
+draft: false
+layout: page
+---
+The present version of TARDIS would not have been possible 
+if not for the work of the following active and past members 
+of our community. To know more about members with specific 
+roles, visit the <a href="../community_roles/">Community Roles</a> page.
+
+## GSoC/SoCIS Students
+A brief explanation on GSoC & SoCIS
```

Comment:
Can you please put a brief explaination of what GSoC and SoCIS are (hyperlinking them to their website)?